### ISSUE TITLE
Include link and short explanation to qa.poole.ch

### ISSUE THREAD

**Issue by stephan75** [IpswichMapper/SwiftAddress#23]
When OSM users want to find adequate areas in order to record house numbers via SwiftAdress, maybe they have one question:

_In what streets or areas is still a lack of housenumbers or buildings in OSM data?_

One good tool among others can be http://qa.poole.ch/ ... see https://wiki.openstreetmap.org/wiki/Qa.poole.ch_(QA_tool) for more information.

So what about mentioning a link or short explanation to that OSM QA tool  inside the SwiftAddress app and in the readme.md in this repo?

**Comment by IpswichMapper** [comment]
yes this seems useful. Certainly seems to work better than OSM inspector

**Comment by IpswichMapper** [comment]
Where would this go in the app? At a welcome screen? In that case another issue to create a "welcome screen" should be made.

Instead, for now I think I'll include it in the github README, OSM WIKI page and a guide if I write one.

**Comment by IpswichMapper** [comment]
> Where would this go in the app? At a welcome screen? In that case another issue to create a "welcome screen" should be made.

> Instead, for now I think I'll include it in the github README, OSM WIKI page and a guide if I write one.

@stephan75 thoughts?

**Comment by stephan75** [comment]
Indeed the best place should be in the app's documentation, where YOU should decide the place: either github readme.md or page in the OSM wiki.
Although the text in the GERMAN wiki page from wiki user Lübeck (who is user osmvermesser in github here, I assume) is a really good base.

Give us a hint when you want to have a translation of these german text parts into english.
